fuse/eval/examples/examples_seq_gen.py

```diff
@@ -47,11 +46,7 @@ def example_seq_gen_0(seed: int = 1234) -> Dict[str, Any]:
         [
             (
                 "perplexity",
-                CI(
```

Comment:
We do cover CI in other examples I hope.
I removed it because the perplexity now is computed at the batch level (more optimized) and I'm not sure it's a metric we would want a CI for.
We can support it back when necessary.